controllers/domain/getDomain.js

```diff
@@ -68,7 +68,7 @@ module.exports = async (req, res) => {
             }
           }
 
-          if (item?.domain?.credderScore >= MIN_CREDDER_SCORE || !CREDDER_CHECK_ENABLED) {
+          if (item?.domain?.credderScore >= CREDDER_MIN_SCORE || !CREDDER_CHECK_ENABLED) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `item?.domain?.credderScore >= CREDDER_MIN_SCORE || !CREDDER_CHECK_ENABLED` might not work as expected. If `CREDDER_CHECK_ENABLED` is `false`, it will push the item regardless of the `credderScore`. This could lead to items with a `credderScore` less than `CREDDER_MIN_SCORE` being added to the data array when `CREDDER_CHECK_ENABLED` is `false`. 

If you want to add an item only when `CREDDER_CHECK_ENABLED` is `false` or when `credderScore` is greater than or equal to `CREDDER_MIN_SCORE`, you should change the order of the conditions.

```diff
- if (item?.domain?.credderScore >= CREDDER_MIN_SCORE || !CREDDER_CHECK_ENABLED) {
+ if (!CREDDER_CHECK_ENABLED || item?.domain?.credderScore >= CREDDER_MIN_SCORE) {
    data.push(item);
  }
```
This way, if `CREDDER_CHECK_ENABLED` is `false`, it won't check the `credderScore` and will just add the item. If `CREDDER_CHECK_ENABLED` is `true`, it will then check if `credderScore` is greater than or equal to `CREDDER_MIN_SCORE` before adding the item.


<!-- This is an auto-generated comment by OSS CodeRabbit -->